app/workers/enqueue_searches_for_external_registry_bikes_worker.rb

```diff
@@ -0,0 +1,15 @@
+class EnqueueSearchesForExternalRegistryBikesWorker < ScheduledWorker
```

Comment:
I think it's good to switch to these separate workers, rather than stuffing them both into the same one.

Can you make the class names the same, pluralization be damned - except for `Enqueue`? It will just make matching things easier for me. so `EnqueueSearchForExternalRegistryBikesWorker` and `EnqueueEmailBikePossiblyFoundNotificationsWorker`

Reply:
I dig it. Updated in 6e643789 and f43621fb